python/lsst/daf/persistence/butler.py

```diff
@@ -39,15 +39,9 @@
 import lsst.pex.logging as pexLog
 import lsst.pex.policy as pexPolicy
 from lsst.daf.persistence import StorageList, LogicalLocation, ReadProxy, ButlerSubset, ButlerDataRef, \
```

Comment:
I'm not actually sure it's safe to import directly from the package a module lives in; it best it seems fragile against moving that module up or down in the list of modules imported in the package's `__init__` file (and I'm actually a bit surprised it works at all).